DATA/production/qc-async/mft.json

```diff
@@ -23,6 +23,28 @@
       }
     },
     "tasks": {
+      "MFTClusters": {
+        "active": "true",
+        "taskName": "Clusters",
+        "className": "o2::quality_control_modules::mft::QcMFTClusterTask",
+        "moduleName": "QcMFT",
+        "detectorName": "MFT",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "The other type of dataSource is \"direct\", see basic-no-sampling.json.",
+        "dataSource": {
+          "type": "direct",
+          "query": "randomcluster:MFT/COMPCLUSTERS/0;clustersrof:MFT/CLUSTERSROF/0;patterns:MFT/PATTERNS/0;cldict:MFT/CLUSDICT/0?lifetime=condition&ccdb-path=MFT/Calib/ClusterDictionary"
+        },
+        "taskParameters": {
+          "maxClusterROFSize" : "5000",
+          "maxDuration" : "60000",
+          "timeBinSize" : "0.1",
+          "ROFLengthInBC" : "198",
+          "geomFileName" : "o2sim_geometry-aligned.root"
```

Comment:
does it mean that you will rely on this file to be present? is it present in the async reco job environment?

Reply:
I am not sure myself, but I checked a recent log file from such a job ( https://alimonitor.cern.ch/users/download.jsp?view=true&path=/alice/data/2022/LHC22o/527108/apass2/0310/o2_ctf_run00527108_orbit0002318592_tf0000000001_epn057/stdout.log ):
```
Calling './async_pass.sh wn.xml ; echo payload-2777879583'
processing run 527108, from period LHC22o with pp collisions and mode remote
./
./o2sim_grp_b0_128HB.root
./o2sim_grp_b2m_128HB.root
./TGeoManager_1671284426276.root
./o2sim_grp_b5p_128HB.root
./o2sim_geometry-aligned.root
./o2sim_grp_b2p_128HB.root
./o2sim_grp_b5m_128HB.root
GRP B field = o2sim_grp_b5p_128HB.root
No runInput_527108.tgz, let's hope we don't need it
Use calib-workflow.sh from O2
Use aggregator-workflow.sh from O2
Checking current directory content
total 21884
-rw-r--r--  1 alicesgm alicesgm     3149 Jul 15 13:48 o2sim_grp_b5p_128HB.root
-rw-r--r--  1 alicesgm alicesgm     3149 Jul 15 13:48 o2sim_grp_b5m_128HB.root
-rw-r--r--  1 alicesgm alicesgm     3149 Jul 15 13:48 o2sim_grp_b2p_128HB.root
-rw-r--r--  1 alicesgm alicesgm     3149 Jul 15 13:48 o2sim_grp_b2m_128HB.root
-rw-r--r--  1 alicesgm alicesgm     3149 Jul 15 13:48 o2sim_grp_b0_128HB.root
-rw-r--r--  1 alicesgm alicesgm  2045579 Oct 22 00:28 CTFs.xml
-rw-r--r--  1 alicesgm alicesgm  5054464 Dec 19 13:38 TGeoManager_1671284426276.root
-rw-r--r--  1 alicesgm alicesgm  5054464 Dec 19 13:40 o2sim_geometry-aligned.root
-rwxr--r--  1 alicesgm alicesgm     1828 Dec 26 07:25 alien_async_pass_v4.sh
-rw-r--r--  1 alicesgm alicesgm    11246 Dec 26 07:26 validation_epn.sh
-rw-r--r--  1 alicesgm alicesgm 10009508 Dec 26 07:30 commonInput.tgz
-rwxr-xr-x  1 alicesgm alicesgm    16633 Jan 10 01:06 async_pass.sh
-rw-r--r--  1 alicesgm alicesgm      569 Jan 11 11:33 logging.properties
drwxr-xr-x 21 alicesgm alicesgm      640 Jan 11 11:33 ..
-rw-r--r--  1 alicesgm alicesgm      786 Jan 11 11:33 access_log
-rw-r--r--  1 alicesgm alicesgm    93871 Jan 11 11:33 wn.xml
-rw-------  1 alicesgm alicesgm     1411 Jan 11 11:33 jobtoken7798557797788274014.pem
-rw-------  1 alicesgm alicesgm     1675 Jan 11 11:33 jobtoken10340312878461203449.pem
-rw-r--r--  1 alicesgm alicesgm        0 Jan 11 11:33 stderr
drwxr-xr-x  3 alicesgm alicesgm      126 Jan 11 11:33 tmp
-rw-r--r--  1 alicesgm alicesgm    16200 Jan 11 11:33 list.list
lrwxrwxrwx  1 alicesgm alicesgm       24 Jan 11 11:33 o2sim_grp.root -> o2sim_grp_b5p_128HB.root
-rwxr-xr-x  1 alicesgm alicesgm     3203 Jan 11 11:33 calib-workflow.sh
-rwxr-xr-x  1 alicesgm alicesgm    14208 Jan 11 11:33 aggregator-workflow.sh
drwxr-xr-x  3 alicesgm alicesgm     4096 Jan 11 11:33 .
-rw-r--r--  1 alicesgm alicesgm    29443 Jan 11 11:33 stdout

```
So apparently it is there. @chiarazampolli is it the plan to keep `o2sim_geometry-aligned.root` there or should we move to the DPL CCDB backend here as well?

Reply:
We should remove the use of this as external file, and get it from CCDB for sure. As you know, I am working on it for TOF and GLO :-)

Reply:
OK, I wasn't sure if that's the same file :) thanks!

@tomas-herman I suppose it is fine to keep it this way for a little bit, but would you be able to retrieve this file using the CCDB query, like you do for the dictionary or with the [GRPGeomHelper](https://github.com/AliceO2Group/QualityControl/blob/master/doc/Advanced.md#access-grp-objects-with-grp-geom-helper)? Laurent did the latter in his [latest PR](https://github.com/AliceO2Group/QualityControl/pull/1608/files).

Reply:
I prefer to keep it like this for this PR and we will make another PR with the switch to using CCDB query in asnyc QC, MC QC and online QC together.